Shopist/Shopist/Scenes/MasterViewController.swift

```diff
@@ -0,0 +1,39 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import UIKit
+
+final class MasterViewController: ListViewController {
```

Comment:
How about naming it `CategoriesListViewController`? "Master" doesn't describe what this screen is responsible for.